bot/bot.py

```diff
@@ -94,7 +94,7 @@ def _recreate(self) -> None:
 
         # Its __del__ does send a warning but it doesn't always show up for some reason.
         if self.http_session and not self.http_session.closed:
-            log.warning(
+            log.info(
```

Comment:
While the consequences of the situation are not that big, I believe it should remain warning because this would happen due to programmer error (i.e. not manually closing before clearing/logging in).